examples/animation/dynamic_image2.py

```diff
@@ -30,6 +30,14 @@ def f(x, y):
 ani = animation.ArtistAnimation(fig, ims, interval=50, blit=True,
                                 repeat_delay=1000)
 
-# ani.save('dynamic_images.mp4')
+# To save the animation, use e.g.
+#
+# ani.save("movie.mp4")
+#
+# or
+#
+# from matplotlib.animation import FFMpegWriter
+# writer = Writer(fps=15, metadata=dict(artist='Me'), bitrate=1800)
```

Comment:
This should be `FFMpegWriter` or the above line should be `... as Writer`.  Same for other files.

Reply:
fixed